composer.json

```diff
@@ -6,22 +6,23 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/allowed_formats": "~1.1",
-        "drupal/core": "~8.6@alpha",
-        "drupal/paragraphs": "~1.2",
+        "drupal/allowed_formats": "^1.1",
+        "drupal/core": "^8.6",
+        "drupal/paragraphs": "^1.3",
         "php": "^7.1"
     },
     "require-dev": {
-        "composer/installers": "^1.2",
-        "cweagans/composer-patches": "~1.0",
-        "drupal-composer/drupal-scaffold": "^2.2",
+        "composer/installers": "~1.5",
+        "cweagans/composer-patches": "~1.6",
+        "drupal-composer/drupal-scaffold": "~2.2",
+        "drupal/coder": "~8.2.12",
         "drupal/config_devel": "~1.2",
-        "drupal/drupal-extension": "^4.0.0@alpha",
-        "drush/drush": "^9",
-        "nikic/php-parser": "~3",
-        "openeuropa/code-review": "~0.3",
-        "openeuropa/drupal-core-require-dev": "~8.6",
-        "openeuropa/task-runner": "~0.5"
+        "drupal/drupal-extension": "~4.0.0@alpha",
+        "drush/drush": "~9.3",
+        "nikic/php-parser": "~3.1.5",
+        "openeuropa/code-review": "~1.0.0-alpha4",
```

Comment:
Same here, the `^` operator should be used if we are specifying the version up to patch level.